kola/tests/misc/selinux.go

```diff
@@ -75,6 +77,46 @@ func testSelinuxCmds(c cluster.TestCluster, m platform.Machine, cmds []cmdCheckO
 	}
 }
 
+// toggleSelinuxBoolean checks the original value of a provided SELinux boolean
+// and returns an array that contains the opposite values
+func toggleSelinuxBoolean(c cluster.TestCluster, m platform.Machine, seBool string) ([3]string, error) {
```

Comment:
The function name seems to imply that it's performing the action of toggling the boolean, maybe `getSelinuxBooleanState`?

Also, let's make a struct for the return contents rather than returning a list of strings. Then we can have names that imply what each field should be explicitly.